Meziantou.FluentAssertionsAnalyzers.Tests/Helpers/ProjectBuilder.cs

```diff
@@ -98,6 +98,8 @@ private ProjectBuilder AddNuGetReference(string packageName, string version, str
     public ProjectBuilder AddMSTestApi() => AddNuGetReference("MSTest.TestFramework", "2.2.10", "lib/netstandard1.0/");
 
     public ProjectBuilder AddNUnitApi() => AddNuGetReference("NUnit", "3.13.3", "lib/netstandard2.0/");

```

Comment:
You can rename this one
```suggestion
    public ProjectBuilder AddNUnit3Api() => AddNuGetReference("NUnit", "3.13.3", "lib/netstandard2.0/");

```